src/pages/Inquiry/Inquiry/Inquiry.module.scss

```
@@ -0,0 +1,187 @@
+$default: #222222;
+$highlight: #ff7f23;
+$back: white;
+$normal: #666666;
+$disabled: #c4c4c4;
+
+.container {
+  width: calc(100vw - 100px);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  margin-left: 100px;
+
+  @media (max-width: 576px) { // 모바일 브레이크 포인트
```

미디어 쿼리 관련 함수 사용 부탁드립니다!
```suggestion
  @use "src/utils/styles/mediaQuery" as media;
  @include media.media-breakpoint-down(mobile) {
```

---

적용했습니다. 3defb199c7f7de89809121f2245a59c8388ebe2f